aten/src/THCUNN/BatchNormalization.cu

```diff
@@ -141,8 +141,8 @@ template <typename Dtype, typename Acctype, typename DeviceTensor1, typename Dev
 __global__ void BatchNormalizationUpdateOutputInference_kernel(
     const DeviceTensor3 input,
     DeviceTensor3 output,
-    DeviceTensor1 runningMean,
-    DeviceTensor1 runningVar,
+    const DeviceTensor1 runningMean,
+    const DeviceTensor1 runningVar,
```

Comment:
Why are you marking these as const?

Reply:
good point! will fix.

Reply:
Oh that's because that this is the inference forward path, which means that the running stats will not be updated.